src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public abstract class Machine {
+    public abstract  void doWork();
+    public abstract  void stopWork();
+
+    }

```

Comment:
There is an unnecessary empty line after the class declaration. According to the checklist, there shouldn't be redundant empty lines after class declaration or method signature.